### ISSUE TITLE
kernel source

### ISSUE THREAD

**Issue by empierre** [mdrjr/odroid-utility#18]
Hello,

  I have a Ralink Technology, Corp. RT5572 Wireless Adapter  (the odroid rapid one) which has a kernel bad support. 

  I would like to compile the module but I need the source. Do you have a package for it ? 

Thanks,

**Comment by mdrjr** [comment]
Its good when you say what board you have...

**Comment by empierre** [comment]
Right, I have a U3 and a XU4, my issue is on the wifi module 4 (as some
others on the forum).

I want to build it from source for it disconnects a lot
